From 695eba9c72ab821dd94257fce155b66582c0e865 Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@stusta.de>
Date: Wed, 29 Aug 2012 20:25:27 +0300
Subject: remove the empty contrib/dist/debian/

It was empty and the up-to-date Debian packaging is anyway
maintained separately in Debian.
---
 configure.ac                    |    1 -
 contrib/dist/Makefile.am        |    2 +-
 contrib/dist/debian/Makefile.am |    1 -
 3 files changed, 1 insertion(+), 3 deletions(-)
 delete mode 100644 contrib/dist/debian/Makefile.am

diff --git a/configure.ac b/configure.ac
index 0ffb31c..6f34968 100644
--- a/configure.ac
+++ b/configure.ac
@@ -495,7 +495,6 @@ Makefile
 
 contrib/Makefile
 contrib/dist/Makefile
-contrib/dist/debian/Makefile
 contrib/dist/gentoo/Makefile
 contrib/dist/redhat/Makefile
 contrib/dist/redhat/mc.spec
diff --git a/contrib/dist/Makefile.am b/contrib/dist/Makefile.am
index 113547a..d521b0c 100644
--- a/contrib/dist/Makefile.am
+++ b/contrib/dist/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = debian gentoo redhat
+SUBDIRS = gentoo redhat
 
 
 AUTOMAKE_OPTIONS = 1.5
diff --git a/contrib/dist/debian/Makefile.am b/contrib/dist/debian/Makefile.am
deleted file mode 100644
index 8b13789..0000000
--- a/contrib/dist/debian/Makefile.am
+++ /dev/null
@@ -1 +0,0 @@
-
-- 
1.7.10.4

